package-lock.json

```diff
@@ -10,7 +10,7 @@
         "./packages/*"
```

Comment:
@abhinav-from-contentstack 
Please push the updated lock file after running bootstrap command

Reply:
@netrajpatel done